openlibrary/plugins/upstream/covers.py

```diff
@@ -1,6 +1,7 @@
 """Handle book cover/author photo upload.
 """
 import web
+import requests
```

Comment:
I believe that `from six.moves import urllib` can now be removed.

Reply:
Yes.